docs/macos_documentation/installation.md

```diff
@@ -0,0 +1,24 @@
+# MacOS Installation Guide
```

Comment:
Thanks for the update! Can we use the following directory instead: docs/installation/macos.md ?

Reply:
Thank you for the recommendation! I've updated the directory.